src/components/Header/Header.jsx

```diff
@@ -58,11 +61,35 @@ export const Header = props => {
     }
   }, []);
 
+  const toggleTheme = (e) => {
+    console.log("navbar toggle click")
+    if (theme === 'dark' ||  e.target.value === "checked") {
+      localStorage.setItem('mode',"light");
+      setTheme('light');
+    } else {
+      localStorage.setItem('mode',"dark");
+      setTheme('dark');
+    }
+  };
+
```

Comment:
@ypan710 Thanks for the updates. Info modals look correct now. However, I am still having an issue with the toggle button. The button works to change the theme and when I refresh the page the toggle is in the correct position. However, it is not actually changing the toggle when I am on the page. See video

I think the issue may have something to do with your `toggleTheme` function. Do you need to set the toggle state using `setChecked` in this function?

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/121642083/e8cc0e8c-cec7-4ca9-a229-98e828af1034